src/Umbraco.Web.BackOffice/Filters/AppendCurrentEventMessagesAttribute.cs

```diff
@@ -14,19 +14,28 @@ namespace Umbraco.Web.WebApi.Filters
     /// </summary>
     internal sealed class AppendCurrentEventMessagesAttribute : ActionFilterAttribute
```

Comment:
Since we're not using this as an attribute and instead as a TypeFilter it can just be an IActionFilter